packages/klevu-ui/src/components/klevu-search-field/klevu-search-field.tsx

```diff
@@ -153,6 +159,10 @@ export class KlevuSearchField {
       searchModifiers.push(sendSearchEvent())
     }
 
+    if (this.usePersonalisation) {
```

Comment:
isnt there a need to check for settings here? maybe the `enablePersonalisationInSearch` flag should be used here

Reply:
This is more generic component and I'm not sure what exactly the `enablePersonalisationInSearch` should affect.